index.html

```diff
@@ -43,9 +43,176 @@ <h3 class="h3">LET’S CONNECT</h3>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
+          <img src="images/SnapshootPortfolio.png" alt="Snapshoot">
+          <div class="title-container">
+            <h2>Multi-Post
+              Stories</h2>
+            <div class="list">
+
+              <span class="list-item-item1">Canopy</span>
+              <span class="list-item">back end Dev</span>
+              <span class="list-item">2015</span>
+            </div>
+          </div>
+          <div class="text-container">
+            <p>A daily selection of privately
+              personalized reads; no accounts or
+              sign-ups required.</p>
+          </div>
+          <div class="languages-list-container">
+            <ul class="lang-container">
+              <li class="langage-item"><span>JavaScript</span></li>
+              <li class="langage-item"><span>HTML</span></li>
+              <li class="langage-item"><span>CSS</span></li>
+            </ul>
+          </div>
+          <button type="button" class="see-btn">See Project</button>
+        </li>
       </ul>
     </div>
   </section>
+  <section class="about-me">
+    <header class="Title">
+      <h1>About Myself</h1>
+    </header>
+    <div class="paragraph">
+      <p>Hello I’m a software developer! I can help you build
+        a product , feature or website Look through some of my work and experience!
+        If you like what you see and have a
+        project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <header class="icons">
+      <h3 class="h3">LET’S CONNECT</h3>
+      <ul class="headline-img">
+        <li>
+          <a href="#"><img src="images/Vector(3).svg" alt="twitter-icon" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Linkedinicon.svg" alt="linkdn-icon" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Vector(2).svg" alt="monster-icon" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Vector(1).svg" alt="gitbubicon" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Vector.svg" alt="thumb" /></a>
+        </li>
+      </ul>
+    </header>
+    <button type="button" class="see-btn">Get my resume</button>
+      <ul>
+        <li class="Languages-arrow-bottom">
+          <h2 class="titles">Languages</h2>
+          <img class="bottom" src="images/arrow_bottom.png" alt="arrow_bottom">
+        </li>
+      </ul>
+    <ul>
+      <li class="Languges-icon">
+        <img class="ellipse" src="images/Ellipse1.jpg" alt="">
+        <h4>JavaScript</h4>
+      </li>
+      <li class="Languges-icon">
+        <img class="ellipse" src="images/Ellipse1(1).jpg" alt="">
+        <h4>HTML</h4>
+      </li>
+      <li class="Languges-icon"> 
+        <img class="ellipse" src="images/Ellipse1(2).jpg" alt="">
+        <h4>CSS</h4>
+      </li>
```

Comment:
Kindly use the proper fonts  in this section like in the template:

<img width="337" alt="image" src="https://user-images.githubusercontent.com/87897753/159583786-53ac2d56-1123-40a5-b855-8db0e8a1443c.png">

<img width="416" alt="image" src="https://user-images.githubusercontent.com/87897753/159583894-7184cc9e-5696-46cc-aae1-fddf6d1ce1fb.png">